shell/common/rasterizer.h

```diff
@@ -401,12 +402,14 @@ class Rasterizer final {
   std::unique_ptr<Surface> surface_;
   std::unique_ptr<flutter::CompositorContext> compositor_context_;
   std::unique_ptr<flutter::LayerTree> last_layer_tree_;
+  std::unique_ptr<flutter::LayerTree> resubmitted_layer_tree_;
```

Comment:
nit: a comment here may be helpful for someone skimming over this header

Reply:
Added a comment to both this and the `last_layer_tree` fields.